services/apiService.ts

```diff
@@ -19,6 +19,10 @@ import {
   BoostClaimParams,
 } from "types/backTypes";
 
+export type QuestDocumentParams = {
```

Comment:
where is this used?

Reply:
i've removed it from my document since it was not used.